### PR TITLE
updated committer and TSC checklist

### PR DESCRIPTION
**Pull request by KipTwitchell**
Signed-off-by: KipTwitchell <kip.twitchell@us.ibm.com>

### CODE REVIEW

File: COMMITTERS.csv [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
+Sandy Peresie,speresie1@yahoo.com,SaPeresi,Python Developer and Spark DM,,X,,,,,,,
 Rakesh Kant,,rkant0721,Apache Spark integration,,X,,,,,,,
 Jeff Horner,,jhornert,Integration architect,,X,X,X,X,X,X,X,X
 Eugene Morrow,,emorrow38,Documentation lead,,,,,X,,,,
-Bob McCormack,bobmcc@au1.ibm.com,bobmcc9,Build lead,,,X,X,X,X,X,X,X
 Jim Hladyshewsky,jim.hladyshewsky@us.ibm.com,JIMHLADYSHEWSKY,Delivery Manager,X,,,,,,,,
 Elana Dunn,,elanadunn,Assistant DM,X,,,,,,,,
 ,,,,,,,,,,,,
 **Repositories**,,,,,,,,,,,,
-Name, Abbreviation for Repo, Purpose ,Status,,,,,,,,,
```
Comment (jmertic) [inline COMMITTERS.csv:15]: Hey @KipTwitchell - could we move the transfer status to a different file, maybe under `tsc`? That would help this file be parsed more consistently. Thanks!

**Comment by KipTwitchell** [comment]
@jmertic Updates done.

**Comment by jmertic** [comment]
Also @KipTwitchell - you need to enable two-factor authentication for your GitHub account. Details at https://docs.github.com/en/github/authenticating-to-github/configuring-two-factor-authentication

**Comment by jmertic** [comment]
@KipTwitchell The teams and access structure should be all in place now. Feel free to merge this in and let me know if any of the permissions aren't correct.

**Pull request merged**